src/hb-ot-layout-gsubgpos.hh

```diff
@@ -114,6 +114,7 @@ struct hb_closure_context_t :
 
   void flush ()
   {
+    hb_set_del_range (output, face->get_num_glyphs (), hb_set_get_max (output));	/* Remove invalid glyphs. */
```

Comment:
What if the same font states higher num glyphs, I think we need this kind of https://github.com/harfbuzz/harfbuzz/commit/baf7779d2d6e4810168a8f036bbf8f9e6493dd1a limitations rather than depending on details fonts still can change.

Reply:
`hb_ot_layout_lookups_substitute_closure ` already has such a limit `HB_CLOSURE_MAX_STAGES` but the current value 32 does not prevent fuzzer from timing out. Are you suggesting to lower it?
My concern about doing so is that it might start rejecting valid fonts.

Throwing out invalid glyphs has no such concern. It is cheap and has no downside. Yes, if a fuzzed font has a large num_glyphs, this check can't prevent it from timing out again. Probably we should do both.

Reply:
We should be highly careful about not rejecting valid fonts, if you don't see another limitation possible for now and see this as last resort, it is ok and please merge it. Thank you very much

Reply:
Meant I am *not* in favor of :D